src/Infrastructure/TimeMgr/src/ESMCI_Time.C

```diff
@@ -830,13 +830,48 @@ namespace ESMCI{
       return(rc);
     }
 
-    if (this->calendar->calkindflag == ESMC_CALKIND_JULIANDAY ||
-        this->calendar->calkindflag == ESMC_CALKIND_MODJULIANDAY ||
-        this->calendar->calkindflag == ESMC_CALKIND_NOCALENDAR) {
-      ESMC_LogDefault.MsgFoundError(ESMC_RC_ARG_WRONG,
-        "; calkindflag is JULIANDAY, "
-        "MODJULIANDAY or NOCALENDAR.", ESMC_CONTEXT, &rc);
-      return(rc);
+    switch (this->calendar->calkindflag)
+    {
+        case ESMC_CALKIND_NOLEAP:
+          ESMC_LogDefault.MsgFoundError(ESMC_RC_ARG_WRONG,
+            "; calkindflag is NOLEAP. Only GREGORIAN and JULIAN are supported.",
```

Comment:
What do you think about adding "when syncing to real time." to the end of the error message?  Just to make it more clear?

Reply:
@oehmke Okay. I add it.